File: node/src/javascript/getArticle/getNewTitle.js
var article = require('./articleClass')

function getNewTitle(articles){
    var titles = articles[0]
    var desc = articles[1]

    articles = []
    for (let i = 0; i < titles.length; i++) {
        articles.append(new article(titles[i], desc[i]))
    }
    console.log(titles)

    var artNum = Math.random()*articles.length
    var randomArticle1 = articles[artNum]
    articles.pop(artNum)
    artNum = Math.random()*articles.length
    var randomArticle2 = articles[artNum]

    var articleTitle = article.combineNounArticlesTitle(randomArticle1, randomArticle2)

    var newTitle = ""
    for (var word in articleTitle.split(' ')){
        if (word.length > 1){
            newTitle += word.slice(0,1).toUpperCase() + word.slice(1) + " "
        }else{
            newTitle += word.slice(0,1).toUpperCase() + " "
        }
    }

    return randomArticle1.title + "☃" + randomArticle2.title + "☃" + newTitle
}

module.exports = getNewTitle

File: python/getNewTitle.py
from webScraping import WebScraping
from ArticleClass import Article
import random

def getNewTitle():
    newInstance = WebScraping()
    titles, desc = newInstance.getTitlesAndDesriptions()

    articles = []
    for i in range(len(titles)):
        articles.append(Article(titles[i], desc[i]))

    randomArticle1 = random.choice(articles)
    articles.remove(randomArticle1)
    randomArticle2 = random.choice(articles)

    articleTitle = Article.combineNounArticlesTitle(randomArticle1, randomArticle2)

    newTitle = ""
    for word in articleTitle.split(' '):
        newTitle += word.capitalize() + " "

    return randomArticle1.title + "☃" + randomArticle2.title + "☃" + newTitle

